src/app/docs/summary-action-bar/summary-action-bar-modal-docs.component.html

```diff
@@ -0,0 +1,70 @@
+<sky-modal>
+  <sky-modal-header>
+    Summary action bar in a modal
+  </sky-modal-header>
+  <sky-modal-content>
+    Hello world!
+  </sky-modal-content>
+  <sky-modal-footer>
+    <sky-summary-action-bar>
+      <sky-summary-action-bar-actions>
+        <sky-summary-action-bar-primary-action
+          (actionClick)="instance.save()"
+        >
+          Save
+        </sky-summary-action-bar-primary-action>
+        <sky-summary-action-bar-secondary-actions>
+          <sky-summary-action-bar-secondary-action>
+            Secondary action
+          </sky-summary-action-bar-secondary-action>
+          <sky-summary-action-bar-secondary-action>
+            Secondary action 2
+          </sky-summary-action-bar-secondary-action>
+        </sky-summary-action-bar-secondary-actions>
+        <sky-summary-action-bar-cancel>
+          Cancel
+        </sky-summary-action-bar-cancel>
+      </sky-summary-action-bar-actions>
+      <sky-summary-action-bar-summary>
+        <div>
+          <sky-key-info>
+            <sky-key-info-value>$123.00</sky-key-info-value>
+            <sky-key-info-label>Field 1</sky-key-info-label>
+          </sky-key-info>
+          <sky-key-info>
+            <sky-key-info-value>$456.00</sky-key-info-value>
+            <sky-key-info-label>Field 2</sky-key-info-label>
+          </sky-key-info>
+          <sky-key-info>
+            <sky-key-info-value>18</sky-key-info-value>
+            <sky-key-info-label>Field 3</sky-key-info-label>
+          </sky-key-info>
+          <sky-key-info>
+            <sky-key-info-value>$123.00</sky-key-info-value>
+            <sky-key-info-label>Field 1</sky-key-info-label>
```

Comment:
This is very minor, but I'd either come up with non-repeated examples (i.e., change values and continue incrementing the label numbers) or limit this to 3 items. The three values repeated three times seems like a bad thing to demo. I can't imagine a consumer actually thinking that we're advocating displaying the same three items three different times, but no reason to test that. ... Also, is this hard coded value/label setup actually a realistic example? Or should we demonstrate pulling data in dynamically? I'm not sure if it's worth the time/effort, but if it's not a difficult thing, it's probably worth making the demo as realistic as possible.

Reply:
Totally agree. Updated with an example I grabbed from the UX specs. Does this feel better?

Reply:
Not sure if I'm missing something or if something didn't get committed, but I still see the same three items repeated three times each.

Reply:
This should be fixed in my latest push.